backend/api/management/commands/ludoteca_dev.py

```diff
@@ -120,6 +123,9 @@ def handle(self, *args, **options):
         print('2. Create users')
         # TODO: Move this to a config file
         reader = pd.read_csv('https://my.api.mockaroo.com/players.json?key=5dec1ef0', header=0, delimiter=',')
+
+        User.objects.create_superuser(username='admin', email='admin@mail.com', password='admin')
```

Comment:
`mail.com` is a real domain, let's use a dummy one instead, just in case we send e-mails by mistake
```suggestion
        User.objects.create_superuser(username='admin', email='admin@example.com', password='admin')
```